tests/test_product_pages_api.py

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: GPL-3.0+
+import pytest
+import requests_mock
+
+from retasc.constants import SUPPORTED_PRODUCTS, UNSUPPORTED_PHASE_ID
+from retasc.product_pages_api import ProductPagesApi
+
+url = "https://pp.rh.com/"
```

Comment:
```suggestion
PP_URL = "https://product_pages.example.com/"
```

Reply:
ack, deleted all the mock set env as well.